lib/exercises.rb

```diff
@@ -2,19 +2,52 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n*m) where n is the length of the strings area and m is the length of the string/num of characters
+# Space Complexity: O(n) where n is the size of the strings array
 
 def grouped_anagrams(strings)
```

Comment:
Since you're sorting each word this is actually O(n m log m)...  Remember sorting is > O(n)